crates/cli/Cargo.toml

```diff
@@ -1,20 +1,20 @@
 [package]
 name = "javy"
-version = "0.1.0"
+version = "0.3.0"
 authors = ["Saúl Cabrera <saulecabrera@gmail.com>"]
 edition = "2018"
-build = "build.rs"
+build = "src/build.rs"
```

Comment:
Why is  this location change needed?

Reply:
I think I understand; it seems that this change is needed because now the build process relies on the `Opt` module; with these new changes the opt-module becomes a build dependency, so instead of including the build script into the src, I'd prefer if we switch the relationship and move all the code from the Opt module inside the build script, keeping the the build script at the root. That will also make it consistent with the other newly-declared build dependencies (`wizer`, `binaryen`, etc)